salt/sshalert/sshalert.py

```
@@ -17,7 +17,11 @@
     ip = os.environ['SSH_CONNECTION'].split(' ')[0]
     user = os.environ['USER']
 
-    whitelisted = redis_shell is not None and redis_shell.exists('sshalert-whitelist:%s' % ip)
+    try:
+        whitelisted = redis_shell is not None and redis_shell.exists('sshalert-whitelist:%s' % ip)
+    except Exception,e:
+        print >> sys.stderr, "Unable to check whitelisted status, assuming whitelisted: ", e
+        whitelisted = True
```

`traceback.print_exc(file=sys.stderr)` is advisable if you're using a catch-all exception handler.